conformance/utils/kubernetes/apply.go

```diff
@@ -161,6 +182,8 @@ func (a Applier) MustApplyObjectsWithCleanup(t *testing.T, c client.Client, time
 				require.NoErrorf(t, err, "error deleting resource")
 			})
 		}
+
+		require.NoErrorf(t, err, "error updating resource")
```

Comment:
maybe we should do this directly after updating ? (I know the below function is doing it like this but I think thats not intentional.